.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -4,6 +4,8 @@ public class OpenAIClient {
         public OpenAIClient(ApiKeyCredential credential, OpenAIClientOptions options);
         public OpenAIClient(ApiKeyCredential credential);
         protected internal OpenAIClient(ClientPipeline pipeline, OpenAIClientOptions options);
+        public OpenAIClient(string apiKey, OpenAIClientOptions options);
```

Comment:
IDK if architects will be happy with this param name - it doesn't need to block this PR but we should get their eyes on it before GA.

Reply:
Let's defer these unrelated updates to a different PR.